tests/Extensions/Abstractions.UnitTests/Activities/ActivityExtensionsTests.cs

```diff
@@ -59,12 +88,15 @@ public void SetObsoleteTransactionId_SetsValue()
 		{
 			Activity activity1 = new Activity("TransactionIdTest1");
 			Activity activity2 = new Activity("TransactionIdTest2");
-			uint id = 117u;
+			uint initialId = 117u;
+			uint updatedId = 219u;
 
-			activity1.SetObsoleteTransactionId(id);
+			activity1.SetObsoleteTransactionId(initialId);
+			activity1.SetObsoleteTransactionId(updatedId);
+			CheckThatKeyNotDuplicated(activity1.Baggage);
 
-			Assert.AreEqual(id, activity1.GetObsolteteTransactionId());
-			Assert.AreNotEqual(id, activity2.GetObsolteteTransactionId());
+			Assert.AreEqual(updatedId, activity1.GetObsolteteTransactionId());
+			Assert.AreNotEqual(updatedId, activity2.GetObsolteteTransactionId());
```

Comment:
That `GetObsolteteTransactionId` has a little typo. Should be `GetObsoleteTransactionId`

Reply:
Oh, thank you for noticing

Reply:
Fixed